lib/round.rb

```diff
@@ -0,0 +1,61 @@
+class Round
+
+  attr_reader :deck,
+              :turns,
+              :correct_turns
```

Comment:
I don't think you need to call a`.correct_Turns` on a round object anyhwere outside of this class. So, for that reason, I'd remove this attr_reader, and just use your instance variable you declared on line 10 when you need to reference that value throughout this class.